ui/raidboss/data/06-ew/raid/p1s.txt

```diff
@@ -0,0 +1,128 @@
+75.5 "True Flare/True Holy" sync / 1[56]:[^:]*:Erichthonios:661[01]:/
+
+# Second flail set
+91.1 "Gaoler's Flail" sync / 1[56]:[^:]*:Erichthonios:6(?:5F[6-F]|60[01]):/
+105.3 "Warder's Wrath" sync / 1[56]:[^:]*:Erichthonios:662A:/
+
+# Tiles 1
+118.5 "Intemperance" sync / 1[56]:[^:]*:Erichthonios:661E:/
+122.6 "--sync--" sync / 1[56]:[^:]*:Erichthonios:65F5:/
+134.2 "Intemperate Torment" sync / 1[56]:[^:]*:Erichthonios:661F:/
+135.3 "First Element" sync / 1[56]:[^:]*:Erichthonios:662[12]:/
+141.4 "Warder's Wrath" sync / 1[56]:[^:]*:Erichthonios:662A:/
+146.4 "Second element" sync / 1[56]:[^:]*:Erichthonios:662[123]:/
+151.6 "Warder's Wrath" sync / 1[56]:[^:]*:Erichthonios:662A:/
+157.3 "Third element" sync / 1[56]:[^:]*:Erichthonios:662[12]:/
+
+# Second tank mechanic
+167.9 "Pitiless Flail of Grace/Pitiless Flail of Purgation" sync / 1[56]:[^:]*:Erichthonios:660[EF]:/
+171.1 "True Flare/True Holy" sync / 1[56]:[^:]*:Erichthonios:661[01]:/
+
+# Cells 1
+188.2 "Shining Cells" sync / 1[56]:[^:]*:Erichthonios:6616:/
+194.5 "--sync--" sync / 1[56]:[^:]*:Erichthonios:65F5:/
+208.6 "Powerful Fire/Powerful Light + Gaoler's Flail" sync / 1[56]:[^:]*:Erichthonios:661[AB]:/
+211.4 "Powerful Fire/Powerful Light + Gaoler's Flail" sync / 1[56]:[^:]*:Erichthonios:661[AB]:/
+224.2 "Pitiless Flail of Grace/Pitiless Flail of Purgation" sync / 1[56]:[^:]*:Erichthonios:660[EF]:/
+227.6 "True Flare/True Holy" sync / 1[56]:[^:]*:Erichthonios:661[01]:/
+229.7 "--sync--" sync / 1[56]:[^:]*:Erichthonios:65F5:/
+243.7 "Powerful Fire/Powerful Light + Gaoler's Flail" sync / 1[56]:[^:]*:Erichthonios:661[AB]:/
+246.5 "Powerful Fire/Powerful Light + Gaoler's Flail" sync / 1[56]:[^:]*:Erichthonios:661[AB]:/
+255.1 "Shackles of Time" sync / 1[56]:[^:]*:Erichthonios:661C:/
+265.3 "Heavy Hand" sync / 1[56]:[^:]*:Erichthonios:6629:/
+270.0 "Inevitable Flame" sync / 1[56]:[^:]*:Erichthonios:6EC1:/
+270.0 "Out of Time" sync / 1[56]:[^:]*:Erichthonios:661D:/
+278.4 "Slam Shut" sync / 1[56]:[^:]*:Erichthonios:6617:/
+289.6 "--sync--" sync / 1[56]:[^:]*:Erichthonios:65F5:/
+
+# Fourfold Shackles
+294.2 "Fourfold Shackles" sync / 1[56]:[^:]*:Erichthonios:6626:/
+303.2 "Chain Pain #1" sync / 1[56]:[^:]*:Erichthonios:662[78]:/
+308.2 "Chain Pain #2" sync / 1[56]:[^:]*:Erichthonios:662[78]:/
+313.2 "Chain Pain #3" sync / 1[56]:[^:]*:Erichthonios:662[78]:/
+318.2 "Chain Pain #4" sync / 1[56]:[^:]*:Erichthonios:662[78]:/
+328.4 "Warder's Wrath" sync / 1[56]:[^:]*:Erichthonios:662A:/
+
+# Tiles 2
+341.6 "Intemperance" sync / 1[56]:[^:]*:Erichthonios:661E:/
+345.7 "--sync--" sync / 1[56]:[^:]*:Erichthonios:65F5:/
+357.5 "Intemperate Torment" sync / 1[56]:[^:]*:Erichthonios:6620:/
+358.6 "First Element" sync / 1[56]:[^:]*:Erichthonios:662[12]:/
+369.6 "Second Element" sync / 1[56]:[^:]*:Erichthonios:662[12]:/
+373.2 "Gaoler's Flail In/Out" sync / 1[56]:[^:]*:Erichthonios:6(?:5F[6-F]|60[01]):/
+376.3 "Gaoler's Flail In/Out" sync / 1[56]:[^:]*:Erichthonios:6(?:5F[6-F]|60[01]):/
```

Comment:
```suggestion
376.3 "Gaoler's Flail Out/In" sync / 1[56]:[^:]*:Erichthonios:6(?:5F[6-F]|60[01]):/
```

As a tiny nit, if it's an A/B and then B/A situation I usually reverse the text on the second one to make that more clear.